src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public String getRandomBall() {
+        Ball ball = new Ball();
+        ColorSupplier color = new ColorSupplier();
+        ball.setNumber(new Random().nextInt(100));
```

Comment:
Don't create a new instance of Random every time you call getRandomBall(). Instead, create a single static instance of Random at the class level and reuse it.